src/main/java/UI/CreateListingPage.java

```diff
@@ -1,84 +1,131 @@
 package UI;
+import Main.Main;
 import forms.CreateListingForm;
+import forms.LoginForm;
 import useCase.createListing.ListingInteractor;
+import useCase.createListing.ListingResponseModel;
 
+import javax.imageio.ImageIO;
 import javax.swing.*;
 import java.awt.*;
 import java.awt.event.ActionEvent;
 import java.awt.event.ActionListener;
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.IOException;
 import java.text.DecimalFormat;
 import java.util.List;
 import java.util.ArrayList;
 public class CreateListingPage extends Page{
+        //private final GridLayout LAYOUT;
+        private JLabel add_your_listing = new JLabel("Add your listing!");;
+        private JLabel listing_title_label = new JLabel("Listing Title:");;
+        private JTextField listing_text = new JTextField();
+        private JLabel price_label = new JLabel("Listing Price:");
+        private JTextField price_text = new JTextField();
+        private JLabel desc_label = new JLabel("Listing Description:");
+        private JTextField desc_text = new JTextField();
+        private JLabel img_label = new JLabel("Images: ");
+        private JLabel img = new JLabel("No image selected");
+        private JButton upload = new JButton("Upload Images");
+        JLabel picLabel = new JLabel();
+        private JButton submit = new JButton("Submit Listing");
+
+        private BufferedImage myPicture = null;
+        private String fileName;
+        private String filePath;
+        private String responseModel;
+        public CreateListingPage() {
+            super("Create Listing Page");
+
+            this.setLayout(null);
+
+            setUpPanel();
+        }
+        public void setUpPanel(){
 
-        public CreateListingPage(String title) {
-            super(title);
-
-            JPanel createListingPanel = new JPanel();
-            createListingPanel.setLayout(null);
-
-            JLabel add_your_listing = new JLabel("Add your listing!");
+            NavBar nb = new NavBar();
+            nb.setLocation(0, 0);
+            nb.setSize(1280, 100);
+            this.add(nb);
```

Comment:
I changed view so that the navBar won't affect your page anymore, so you might want to remove the navBar from your page. It shouldn't change the implementation but will be cleaner.